src/iobuf.rs

```diff
@@ -755,9 +754,9 @@ pub trait Iobuf: Clone + Show {
   /// let mut tgt3 = [ 0x00, 0x00, 0x00 ];
   ///
   /// assert_eq!(b.peek(0, &mut tgt4), Ok(()));
-  /// assert_eq!(tgt4.iter().map(|&x| x).sum(), 10);
+  /// assert_eq!(tgt4.iter().map(|&x| x).fold(0,|a, b| a + b), 10);
```

Comment:
Ha.   sum is part of a deprecated library. I'm not sure if they plan to replace the lib or move sum.  Either way I'm pretty sure fold isn't going anywhere for a while.